### PR TITLE
Jincheng dev

### PR DESCRIPTION
**Pull request by TownesZhou**
Basic scheduling algorithm completed
Pull request to master branch

### CODE REVIEW

File: ScheduleMaker/doc/allclasses-noframe.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+<script type="text/javascript" src="jquery/jszip-utils/dist/jszip-utils.min.js"></script>
+<!--[if IE]>
+<script type="text/javascript" src="jquery/jszip-utils/dist/jszip-utils-ie.min.js"></script>
+<![endif]-->
+<script type="text/javascript" src="jquery/jquery-1.10.2.js"></script>
+<script type="text/javascript" src="jquery/jquery-ui.js"></script>
+</head>
+<body>
+<h1 class="bar">All&nbsp;Classes</h1>
+<div class="indexContainer">
```
Comment (jillianKhoo) [inline ScheduleMaker/doc/allclasses-noframe.html:22]: nit: can we indent some of these nested components? everything inside indexContainer should be indented so we can see the structure

File: ScheduleMaker/src/api/Schedule.java [hunk 2]
```diff
@@ -0,0 +1,435 @@
...
+	private Instant startTime;
+	private Instant endTime;
+	private boolean [] onDay;
+	private String location;
+	
+	// public method for checking whether two sessions are in time conflict
+	public static boolean isConflict(Session session1, Session session2) {
+		for (int i = 0; i < 5; i++) {
+			if (session1.getOnDay()[i] && session2.getOnDay()[i]) {
+				//if (session1.getStartTime().isAfter(session2.getStartTime()) && session1.getStartTime().isBefore(session2.getEndTime())
```
Comment (jillianKhoo) [inline ScheduleMaker/src/api/Schedule.java]: nit: lets remove these comments

File: ScheduleMaker/src/api/ScheduleTest.java [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+		//schedule.addSession(1, "29905R", "14:00", "15:50", new boolean[] {false, true, false, false, false}, "SAL109");
+		schedule.addSession(1, "29907R", "16:00", "17:50", new boolean[] {false, true, false, false, false}, "SAL109");
+		schedule.addSession(1, "29932R", "11:00", "1:50", new boolean[] {false, true, false, false, false}, "SAL109");
+		schedule.addSession(2, "30025R", "19:00", "20:50", new boolean[] {false, false, false, false, true}, "SAL109");
+		//schedule.addSession(3, "29947D", "10:00", "11:50", new boolean[] {true, false, true, false, false}, "VPD105");
+		//schedule.addSession(3, "29953D", "14:00", "15:20", new boolean[] {true, false, true, false, false}, "ZHS152");
+		schedule.addSession(3, "30273D", "12:00", "13:50", new boolean[] {true, false, true, false, false}, "SLH102");
+		schedule.addSession(4, "29929R", "16:00", "17:50", new boolean[] {false, false, false, false, true}, "GFS116");
+		schedule.addSession(5, "30027R", "19:00", "20:50", new boolean[] {false, false, true, false, false}, "GFS116");
+
```
Comment (jillianKhoo) [inline ScheduleMaker/src/api/ScheduleTest.java:99]: nit: remove the commented out stuff

**Review by jillianKhoo — APPROVED** [review]
Looks good to me, just a few small comments about removing things that you commented out for cleanliness but go ahead and merge other than that. Oh also get rid of the merge conflict in your README

File: ScheduleMaker/src/api/ScheduleTest.java (hunk 3 above)
Comment (TownesZhou) [reply ScheduleMaker/src/api/ScheduleTest.java:99]: Comments removed

**Pull request merged**